script/Ciel/Day 13/13_01_02_CIEL13_7.txt

```diff
@@ -391,12 +451,12 @@ However.
 [sha:6e9fc0f78cd6ed1ce2582c9be45ec0f751693400]{
 -- Page 230, Offset 35916.
 -- 降りそそぐ鉄の雨は、横合いから放たれた鉄の砲弾によって弾かれた。
-The downpour of iron is repelled by a metal shell fired from the side.
+The downpour of iron rain is deflected by a metal shell that comes flying in from the side.
```

Comment:
"downpour of iron rain" is a bit too redundant.
"shell" is a bit hard to visualize as a metaphor.
>The downpour of steel is deflected by a metal missile that comes flying...